### DISCUSSION TITLE
Ability to keep slack integration but remove 'log in with slack' from the login screen.

### DISCUSSION THREAD

**Discussion by broken-wheel** [outline/outline#7116] [Ideas]
I have a self-hosted version. I have OIDC login and slack integration. However, I want to show *only* the OIDC login and not the 'log in with Slack' in the login screen. 
 
I found a related issue: https://github.com/outline/outline/issues/6132

**Comment by Lyushen** [comment]
Could be done by commenting
```
SLACK_CLIENT_ID
SLACK_CLIENT_SECRET
```
additionally to
```
SLACK_VERIFICATION_TOKEN
SLACK_APP_ID
SLACK_MESSAGE_ACTIONS
```

**Comment by penguinsam** [comment]
> I have a self-hosted version. I have OIDC login and slack integration. However, I want to show _only_ the OIDC login and not the 'log in with Slack' in the login screen.   I found a related issue: #6132

I own a self-hosted version with OIDC login too, but cannot make slack integration work. How did you do that? Do you have an qualified FQDN?
